Quaver.Shared/Screens/Select/SelectScreen.cs

```diff
@@ -351,6 +363,32 @@ private static void HandleKeyPressTab()
                 ConfigManager.LeaderboardSection.Value = LeaderboardType.Local;
         }
 
+        /// <summary>
+        ///     Handles when the user presses the del key
+        /// </summary>
+        private void HandleKeyPressDel()
+        {
+            if (!KeyboardManager.IsUniqueKeyPress(Keys.Delete) || DialogManager.Dialogs.Count > 0)
+                return;
+
+            // osu! loaded map check to avoid errors due to the map loaded being virtual from osu.
+            if (MapManager.Selected.Value.Game == MapGame.Osu)
```

Comment:
Use `!= Quaver` here for all cases.

Reply:
This check should also be moved to the method that handles deletion, so it can be reused.